toy-submission/main.py

```diff
@@ -39,29 +36,27 @@
 # Configure the logging module
 logging.basicConfig(level=logging.INFO)
 
-quantize = "llm.int8"
+quantize = "bnb.nf4-dq"  # 4-bit NormalFloat with Double-Quantization (see QLoRA paper)
+checkpoint_dir = Path("checkpoints/openlm-research/open_llama_3b")
```

Comment:
I changed the 7b model for the 3b as it will be faster to prepare and run for the sake of development iteration. I expect that all submitters will use a larger model than 7b anyways.